sympy/simplify/tests/test_radsimp.py

```diff
@@ -393,6 +394,8 @@ def test_fraction():
     assert fraction(exp(n)) == (1, exp(-n))
     assert fraction(exp(-n)) == (exp(-n), 1)
 
+    p = symbols('p', positive=True)
+    assert fraction(exp(-p)*log(p), exact=True) == (exp(-p)*log(p), 1)
```

Comment:
* I referred to the docstring while making this PR. Since the docstring only refers to the case when `exact` is `unset`. It doesnt talk about if `exact=True`.

* The way I improved `exact=True` case is that it now applies to unknown exponents whose values is not definite. 

* I think we should add docstring for `exact=True` case keeping the previous docstrings same.